recipes/python-periodictable/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-periodictable" %}
+{% set version = "1.4.1" %}
+{% set sha256 = "f42e66f6efca33caec4f27dad8d6a6d4e59da147ecf5adfce152cb84e7bd160b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
Looking at the setup.py, it looks like you need `numpy` and `pyparsing` as run requirements